src/main/resources/migrations/00043_add_payouts_foreign_key_to_transaction.sql

```diff
@@ -0,0 +1,5 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:add_payouts_foreign_key_to_transaction
+ALTER TABLE transaction ADD COLUMN gateway_payout_id VARCHAR(26) REFERENCES payouts(gateway_payout_id);
```

Comment:
We also need to create index on this column to be able to search for transactions for a given payout but can be a separate PR.

Reply:
ah, we will have to change the length in the payouts table too then